### PR TITLE
cyclow implementation of dropdown demo

### PR DESCRIPTION
**Pull request by ghost**
At last, my cyclow implementation of dropdown demo. Tested on node 7.7.3 and npm 4.4.1.

The strange error "Couldn't find preset "es2015" relative to directory" was solved patching presets on webpack.config.js. See https://github.com/babel/gulp-babel/issues/93 for more information.

Finally, onchange event gives me enough information to get the selected index. So I don't need any special abstraction. I don't need a new cyclow realease.

We have all cyclow implementations now (for all demos)!

### CODE REVIEW

File: demos/dropdown/cyclow/es6/src/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'in.init': () => state => ({goods, chosen: goods[0]}),
+    'dom.select': index => state => ({...state, chosen: goods[index]})
+  },
+  view: ({goods, chosen}) => ({
+    attrs: {className: {vegetable: 'veggie', fruit: 'fruit'}[chosen.type]},
+    content: [
+      'I postit that ',
+      {
+        tag: 'select',
+        on: {change: (e, next) => next({select: e.target.selectedIndex})},
```
Comment (krawaller) [inline demos/dropdown/cyclow/es6/src/dropdown.js:16]: Thought - is the event handler here bound to the target element, as in regular HTML? If so, maybe more idiomatic to do `this.selectedIndex`?

File: demos/dropdown/cyclow/es6/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: {
+          presets: [
+            'babel-preset-es2015',
+            'babel-preset-stage-0'
+          ].map(require.resolve),
```
Comment (krawaller) [inline demos/dropdown/cyclow/es6/webpack.config.js:16]: This is where you just throw your hands in the air and say whatEVerrrr! :)

File: demos/dropdown/cyclow/es6/src/dropdown.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        content: goods.map(good => ({tag: 'option', content: good.name}))
+      },
+      ` ${chosen.quote}`
```
Comment (krawaller) [inline demos/dropdown/cyclow/es6/src/dropdown.js:19]: This could be just `chosen.quote`, right?

**Comment by krawaller** [comment]
Yeah, I realise now that my explanation text for the dropdown demo is rather weird (and actually incorrect). 

Agreed that no extra abstraction in the framework is strictly needed! In fact, doing it this way and simply ignoring the `value` attribute of the `option` element is much easier. 

Wow, that preset patching thing is so weird. Read the issue and some more background, but, yeah. Weird. Glad you solved it though! :)

File: demos/dropdown/cyclow/es6/src/dropdown.js (hunk 3 above)
Comment (ghost) [reply demos/dropdown/cyclow/es6/src/dropdown.js:19]: right, but for the the whitespace. You can code `' ' + chosen.quote` too.

Comment (krawaller) [reply demos/dropdown/cyclow/es6/src/dropdown.js:19]: oh, missed the space, sorry!

File: demos/dropdown/cyclow/es6/src/dropdown.js (hunk 1 above)
Comment (ghost) [reply demos/dropdown/cyclow/es6/src/dropdown.js:16]: I tried but it doesn't work. It seems Snabbdom doesn't bind `this` with the node or dom element.

File: demos/dropdown/cyclow/es6/webpack.config.js (hunk 2 above)
Comment (ghost) [reply demos/dropdown/cyclow/es6/webpack.config.js:16]: It's weird but it works!

File: demos/dropdown/cyclow/es6/src/dropdown.js (hunk 1 above)
Comment (krawaller) [reply demos/dropdown/cyclow/es6/src/dropdown.js:16]: oooh, it is Snabbdom! I see!

...and now I remember you've already told me that :)

Using the event object is nicer anyway, it feels less magical to me

Comment (ghost) [reply demos/dropdown/cyclow/es6/src/dropdown.js:16]: Of course you could code something like this:
```
on: {change: selectChangeEvent('select')}
```
where 
```
const selectChangeEvent = name => (e, next) => next({[name]: e.target.selectedIndex})
```
maybe I'll add some helpers like this to cyclow later... Sometimes you are not in the mood to think about HTML events. But it's just one-line-helper-function! I like that.

**Pull request merged**